### DISCUSSION TITLE
Is it possible to get each name of styles as a string from iTools Prompt Styler?

### DISCUSSION THREAD

**Discussion by lycos2019** [MohammadAboulEla/ComfyUI-iTools#8] [Q&A] [answered]
![please](https://github.com/user-attachments/assets/e2d91633-b86b-4fe3-aa75-5b1e6ec15fc8)
Since I don't know what all those styles look like I'd like to batch with increment value on template name so that I can see what they look like all at once. but the problem here I can't get each string set for the style name from iTools prompt styler.
I can get a text after prompt styler then texts are all mixed, I can't get the name of style from there.

**Comment by MohammadAboulEla** [comment, marked as answer]
I will try to work on that isa, but for now if you used iTools Prompt Styler extra it return the selected template name, and you can use it with one style and ignore the other 3 styles and you will get the template name used
